src/App.js

```diff
@@ -1,41 +1,60 @@
 import React from 'react'
-import ReactDOM from 'react-dom'
-import { Button } from 'reactstrap'
-import {
-  ButtonDropdown,
-  DropdownToggle,
-  DropdownMenu,
-  DropdownItem
-} from 'reactstrap'
+import ToggleButton from './components/ToggleButton'
+//import CollapseButton from './components/CollapseButton'
+import styled from 'react-emotion'
+
+const Grid = styled('div')`
+  display: grid;
+  grid-gap: 10px;
+`
 
 export default class Example extends React.Component {
   constructor(props) {
     super(props)
 
     this.toggle = this.toggle.bind(this)
     this.state = {
-      dropdownOpen: false
+      buttons: [
+        { text: 'Gazpacho', isSelected: false, id: 1 },
```

Comment:
Du könntest dir auch mit `npm i uid` die Funktion `uid` installieren und oben importieren.
Dann schreibst du anschließend in jedem Button `id: uid()` anstatt manuell Zahlen zu vergeben.